eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/service/DbsnpClusteredHumanVariantAccessioningService.java

```diff
@@ -61,4 +67,27 @@ public DbsnpClusteredHumanVariantAccessioningService(@Qualifier("humanActiveServ
             return Collections.emptyList();
         }
     }
+
+    public Optional<List<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>>> getByIdFields(
+            String assembly, String contig, long start, VariantType type) {
+        List<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>> clusteredVariants =
+                new ArrayList();
+
+        IClusteredVariant clusteredVariantToSearch = new ClusteredVariant(assembly, 0, contig, start, type, false,
+                                                                          null);
+
+        List<AccessionWrapper<IClusteredVariant, String, Long>> activeClusteredVariantWrappers =
+                humanService.get(Collections.singletonList(clusteredVariantToSearch));
+        List<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>> activeClusteredVariants =
+                activeClusteredVariantWrappers.stream()
+                                              .map(v -> new AccessionResponseDTO<>(v, ClusteredVariant::new))
+                                              .collect(Collectors.toList());
+        clusteredVariants.addAll(activeClusteredVariants);
```

Comment:
Intellij suggests ``List<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>> clusteredVariants =
                new ArrayList(activeClusteredVariants);`` and I think that is a good idea too...

Reply:
I kind of like better using the `addAll` because we are adding `activeClusteredVariants` and `activeClusteredVariantsInOperations` to the `clusteredVariants` List, so for me it's easier to read if we use the same method to add in both cases. What do you think about it @sundarvenkata-EBI ?

Reply:
Sure. It was just a judgement call... I am fine with leaving this as it stands...